### PR TITLE
feat: added config, tests and debug for config

### PR DESCRIPTION
**Pull request by Aleksey28**
Completed tasks:
1. Created files src/config.js and test/config-test.js
2. Exported host and port from config.js
3. Imported in several varieties config.js to config-test.js
4. Added test for the imported values
5. Connected the dotenv
6. Connected the debug and debug's points

### CODE REVIEW

File: src/config.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
+import { config } from 'dotenv';
+import debug from 'debug';
+
+config();
+
+const debugConfig = debug('config');
+
+const { PORT = '1337', HOST = 'localhost' } = process.env;
+
+//You need to tap $env:DEBUG="config" in terminal to turn on debug
```
Comment (AndreyBelym) [inline src/config.js]: Nice to see that you use English for comments 😄 

Could you please also add `NOTE:`, `HACK:`or `TODO:`before an actual description? We follow this convention in the TestCafe code.

**Review by AndreyBelym — COMMENTED** [review]
Wow, quite impressive work! Just some questions and nitpicks for now 😄

File: src/config.js (hunk 1 above)
Comment (Aleksey28) [reply src/config.js]: Thank you)
Did it

File: src/config.js [hunk 2]
```diff
@@ -0,0 +1,25 @@
+import { config } from 'dotenv';
+import debug from 'debug';
+
+config();
+
+const debugConfig = debug('config');
+
+const { PORT = '1337', HOST = 'localhost' } = process.env;
+
+//HACK: You need to tap $env:DEBUG="config" in terminal to turn on debug
```
Comment (AndreyBelym) [inline src/config.js]: I don't feel that it's a hack. It's the normal way to enable debugging, so it's just a note 😄

File: src/config.js [hunk 3]
```diff
@@ -0,0 +1,25 @@
+import { config } from 'dotenv';
+import debug from 'debug';
+
+config();
+
+const debugConfig = debug('config');
```
Comment (AndreyBelym) [inline src/config.js]: It's better to get used to do not directly use any primitive, like strings and numbers, except (zeros, empty strings, and one/minus one in some cases). Let's always use constants for primitives and try to find a meaningful name (e.g. `DEBUG_PREFIX_CONFIG`).

BTW, `debug` conventions are that you use your app name as the top-level debug prefix. E.g. we can use the package name. So I expect two constants here 😉

File: src/config.js [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+    PORT,
+    HOST
+};
+
+export default myConfig;
+
+export {
+    PORT,
+    HOST,
+    myConfig as config,
```
Comment (AndreyBelym) [inline src/config.js]: Exporting configuration as `config`here is a bit redudant here (however, it shows your deep understanding of how JS objects are exported 👍 )

I asked you to allow different **imports**, including importing the whole configuration as `config`. A tip here: you can import the default export as a named import :wink: I want to see that in tests. Feel free to send me a personal message if you stuck, don't be shy about that

File: test/config-test.js [hunk 5]
```diff
@@ -0,0 +1,51 @@
+import defaultConfig, { config, HOST, PORT } from '../src/config.js';
+import { expect } from 'chai';
+
+
+describe('Checking "HOST"', () => {
```
Comment (AndreyBelym) [inline test/config-test.js:5]: A very comprehensive test. Excellent work 👍

**Review by AndreyBelym — COMMENTED** [review]
Just some advanced topics

**Comment by AndreyBelym** [comment]
Also, could you please **always** check that your pull requests have a description? 😉

File: src/config.js (hunk 2 above)
Comment (Aleksey28) [reply src/config.js]: Ok) It was just funny to use HACK)

**Review by AndreyBelym — COMMENTED** [review]
Impressive work! Just please open a second pull request, so we have something to play with rebasing 😉

**Review by AndreyBelym — APPROVED** [review]

**Pull request merged**